README.md

```diff
@@ -60,6 +62,7 @@ const saml = new SAML(options);
 - `racComparison`: Requested Authentication Context comparison type. Possible values are 'exact','minimum','maximum','better'. Default is 'exact'.
 
 - `forceAuthn`: if set to true, the initial SAML request from the service provider specifies that the IdP should force re-authentication of the user, even if they possess a valid session.
+- `passive`: if set to true, specifies that the IdP must not visibly take control of the user interface and interact with the user.
```

Comment:
I might be missing it, but I don't see a test for this.

Reply:
No test for `forceAuthn` as well, although only `passive` was absent from the documentation. I've added a new one for both.